js/framework/utils.js

```diff
@@ -41,11 +41,13 @@ function set_hand_cursor_on_widget(widget) {
 
     let enter_id = widget.connect('enter-notify-event', function (widget) {
         let cursor = Gdk.Cursor.new_from_name(widget.get_display(), "pointer");
-        widget.window.set_cursor(cursor);
+        if (widget.window)
```

Comment:
These kinds of "NULL checks" make me nervous - why did we get here with widget.window == NULL?  Is it legitimate to be in this path in this state, or is this hiding a bug from somewhere else?

Also, it feels like only this hunk is directly described by the oneline commit log, I think maybe this should be split?

Reply:
Oh, yeah, this was totally added in a mad rush of adding null checks around get_window. Not needed here. Thanks for noticing! I'll remove this commit when I squash. (I think there was another more legitimate null check in here originally, but that may have found its way to master at some point).